go.mod

```diff
@@ -1,15 +1,23 @@
 module github.com/aws/aws-xray-daemon
```

Comment:
I changed the GO and golang.org/x/text version manually. The other changes were auto-generated by go mod tidy. That includes the additional require block.